src/Interpreters/executeQuery.cpp

```diff
@@ -419,6 +419,9 @@ void logQueryFinish(
         {
             Progress p;
             p.incrementPiecewiseAtomically(Progress{ResultProgress{elem.result_rows, elem.result_bytes}});
+
+            UInt64 real_time_microseconds = (*info.profile_counters)[ProfileEvents::RealTimeMicroseconds];
```

Comment:
you need to define `ProfileEvents::RealTimeMicroseconds` as extern:
```
namespace ProfileEvents
{
    extern const Event RealTimeMicroseconds;
}
```

Reply:
Thanks